src/dealWizard/wizardManager.ts

```diff
@@ -0,0 +1,116 @@
+import { autoinject } from "aurelia-framework";
+import { PLATFORM } from "aurelia-pal";
+import { RouteConfig } from "aurelia-router";
+import { WizardService, IWizardState, IWizardStage } from "services/WizardService";
+import { DealRegistrationData, IDealRegistrationData } from "entities/DealRegistrationData";
+import { IStageMeta, WizardType, STAGE_ROUTE_PARAMETER } from "./dealWizardTypes";
+
+const proposalStage: IWizardStage = {
```

Comment:
I think would be better to scope these routes inside the class

Reply:
Sure, I don't see much difference, but I can move it